test/terra/noise/test_quantum_error.py

```diff
@@ -283,8 +283,9 @@ def depol_error(self, param, **kwargs):
         )
 
     @staticmethod
-    def aslist(qargs):
-        return [q.index for q in qargs]
+    def qubits_list(circ, qargs):
+        qubit_map = {qubit: i for i, qubit in enumerate(circ.qubits)}
+        return [qubit_map[q] for q in qargs]
```

Comment:
`return [circ.find_bit(q).index for q in qargs]` is a one-liner that does the same thing for Terra 0.19+.

Reply:
Cool, didn't know this was added

Reply:
yeah, turns out we had that `qubit_map` line in like 1,000 places, so it was starting to become really necessary.  We still need to add something similar to DAGCircuit, because like 80% of the transpiler passes have something similar as well.